### PR TITLE
feat(*): add rollback to a release version

### LINKED ISSUE #1244: Rollback to a specific version
Allow specifying a release version on `helm rollback`

ref: #1004

### PR DESCRIPTION
**Pull request by adamreese**
closes #1244

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/helm/1248)

### CODE REVIEW

File: cmd/helm/rollback.go [hunk 1]
```diff
@@ -50,8 +53,15 @@ func newRollbackCmd(c helm.Interface, out io.Writer) *cobra.Command {
...
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := checkArgsLength(len(args), "release name"); err != nil {
-				return err
+			switch {
+			case len(args) == 0:
+				return errors.New("This command needs a release name")
+			case len(args) > 1:
+				v, err := strconv.Atoi(args[1])
+				if err != nil {
+					return fmt.Errorf("release version must be an int: %s", err)
```
Comment (technosophos) [inline cmd/helm/rollback.go]: Did we decide we were going to call this `revision` in user-facing docs?

Comment (adamreese) [inline cmd/helm/rollback.go]: I think so?  We didn't really decide

**Comment by adamreese** [comment]
I added the version/revision as an argument but noticed on `helm get` it is a flag.  Probably should be consistent

**Comment by adamreese** [comment]
@technosophos changed to a flag

**Comment by technosophos** [comment]
Double-LGTM, @adamreese 🚢

**Pull request merged**